db/migrations/V1_migration_entity.sql

```diff
@@ -0,0 +1,10 @@
+create table if not exists migrations (
```

Comment:
I thought SQ itself managed the table creation for us? Shouldn't this table be made because its in the GraphQL schema already and isn't prepopulated with data here?

Reply:
There was an instance in the past where the table wasn't created on its own, that's when I started adding new tables as well. I think this happened when we added subquery_versions table. Or maybe that was because of the async execution of the SQLs that we do before running the jest node for SQ. Anyways, I wanted to add a migration to start the sequencing and this does the work, so I'm keeping this for now.